sdk/core/azure-core-test/src/main/java/com/azure/core/test/utils/TestProxyManager.java

```diff
@@ -53,14 +55,12 @@ public void startProxy() {
         try {
             String commandLine = "test-proxy";
             // if we're not running in CI, construct the local path. TF_BUILD indicates Azure DevOps. CI indicates Github Actions.
-            if (Configuration.getGlobalConfiguration().get("TF_BUILD") == null
-                && Configuration.getGlobalConfiguration().get("CI") == null) {
+            if (runningLocally()) {
                 commandLine = Paths.get(TestProxyDownloader.getProxyDirectory().toString(),
                     TestProxyUtils.getProxyProcessName()).toString();
             }
 
-            ProcessBuilder builder = new ProcessBuilder(commandLine, "--storage-location", recordingPath.getPath(), "--", "--urls", getProxyUrl().toString())
-                .directory(TestProxyDownloader.getProxyDirectory().toFile());
+            ProcessBuilder builder = new ProcessBuilder(commandLine, "--storage-location", recordingPath.getPath(), "--", "--urls", getProxyUrl().toString());
```

Comment:
Should the directory be added back but `TestProxyDownloader.getProxyDirectory()` changes based on being in CI or local?